app/views/albums/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1><%= @album.name %>
+<small>Recorded by: <%= @album.artist %></small></h1>
```

Comment:
This line should be indented so that it's clear that it's still part of the `<h1>`